### PR TITLE
feature/adding catalan json file for AmettlerOrigen

### PR DESCRIPTION
**Pull request by cesarocampov**
#### What problem is this solving?

We are adding the catalan language. This is a requirement from a live customer AmetllerOrigen who is having issues on their minicart today. 

#### How to test it?

[Workspace](https://messages--ametllerorigen.myvtex.com/)

### CODE REVIEW

File: messages/ca.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+  
+{
+  "store/product-list.pricePerUnit": "{price} {perMeasurementUnit}",
+  "store/product-list.pricePerUnit.measurementUnit": "per {unitMultiplier} {measurementUnit}.",
+  "store/product-list.availableItems": "{quantity, plural, one {# producte disponible} other {# productes disponibles}}",
+  "store/product-list.unavailableItems": "{quantity, plural, one {# producte no disponible} other {# productes no disponibles}}",
+  "store/product-list.message.cantBeDelivered": "Aquest producte no es pot lliurar a l'adreça proporcionada.",
```
Comment (camilavcoutinho) [inline messages/ca.json]: "Aquest producte no es pot lliurar a l'adreça facilitada."

File: messages/ca.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "store/product-list.message.noLongerAvailable": "Aquest producte no està disponible actualment.",
```
Comment (camilavcoutinho) [inline messages/ca.json]: "Aquest producte ja no està disponible.",

File: messages/ca.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  "store/product-list.message.remove": "retirar",
```
Comment (camilavcoutinho) [inline messages/ca.json]: "suprimeix",

File: messages/ca.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  "store/product-list.quantity-selector.remove": "Retirar",
```
Comment (camilavcoutinho) [inline messages/ca.json]: "Suprimeix",

File: messages/ca.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+  "store/product-list.quantity-label": "{quantity} un.",
+  "store/product-list.delete-button.default-label": "Eliminar Producte",
```
Comment (camilavcoutinho) [inline messages/ca.json:12]: "Suprimeix el producte",

File: messages/ca.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  "store/product-list.priceChanged": "Modificat",
+  "store/product-list.priceOptions": "Opcions",
```
Comment (camilavcoutinho) [inline messages/ca.json]: "Opcions de preu",

File: messages/ca.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+  "store/product-list.changePrice": "Canvi de preu",
+  "store/product-list.revertToOriginal": "Tornar a l'original",
```
Comment (camilavcoutinho) [inline messages/ca.json]: "Torna a l'original",

File: messages/ca.json [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+  "store/product-list.originalPrice": "Preu original",
+  "store/product-list.changeTo": "Canviar a",
+  "store/product-list.manualPricePlaceholder": "Introduïu un valor monetari",
```
Comment (camilavcoutinho) [inline messages/ca.json]: "Introdueix un valor monetari",

File: messages/ca.json [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+  "store/product-list.applyManualPrice": "Aplicar",
```
Comment (camilavcoutinho) [inline messages/ca.json]: "D’acord",

File: messages/ca.json [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+  "store/product-list.quantitySelector.label": "{quantity} {measurementUnit}",
+  "store/product-list.quantitySelector.maxValueLabel": "{quantity} {measurementUnit} +",
+  "store/product-list.quantityUnitMultiplierMismatch": "Aquest producte es ven per fraccions de {unitMultiplier}{measurementUnit}. Per tant, la quantitat ingressada s'ha arrodonit a {roundedValue}{measurementUnit}.",
```
Comment (camilavcoutinho) [inline messages/ca.json]: "Aquest producte es ven per fraccions de {unitMultiplier}{measurementUnit}. Per tant, la quantitat introduïda s'ha arrodonit a {roundedValue}{measurementUnit}.",

File: messages/ca.json [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+  "store/product-list.quantityStepper.label": "Quantitat de productes",
+  "store/product-list.quantityStepper.increaseQuantity": "Augmentar la quantitat",
+  "store/product-list.quantityStepper.decreaseQuantity": "Quantitat decreixent"
```
Comment (camilavcoutinho) [inline messages/ca.json]: "Reduir la quantitat"

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]

File: messages/ca.json (hunk 5 above)
Comment (gabrielyaquinta) [reply messages/ca.json:12]: I think this should stay as "Eliminar"

File: messages/ca.json (hunk 1 above)
Comment (cesarocampov) [reply messages/ca.json]: hi @camilavcoutinho we created a new PR to fix all these comments:
https://github.com/vtex-apps/product-list/pull/96

**Pull request closed without merging**